exec.py

```diff
@@ -171,6 +178,7 @@ def run_help():
     print('  --tiles-coords: needs to be an array of arrays (each of len 3). Defaults to [[66, 43, 7]].')
     print('                  You can find coords by using http://tools.geofabrik.de/calc/?grid=1')
     print('                  Used in load-db(-france) command.')
+    print('  --namespace   : set the --project option of docker-compose, allowing to change name prefix of all used docker images')
```

Comment:
You have the right to make newlines like it's done above. :)

Reply:
Also, should we consider using an argument parsing library ? This script is not that small :o

Also, there is about 1000 lines of python in this repo, this could be worth using the same tooling as for Idunn? (I'm think of black, I don't know if there might be other convenient tools)

Reply:
The code is pretty simple and could be reduced easily for most of it (a lot of the code is repeating itself). I can do it if you want? I'd prefer avoiding adding dependencies for something this "simple" if possible.

Reply:
I feel like this could be worth to do it some day :)

Reply:
> I'd prefer avoiding adding dependencies for something this "simple" if possible.
I just noticed that argparse is part of standard library, I wasn't aware of that :) https://docs.python.org/3.8/library/argparse.html